From 8f7261d27d2c8af674d628af7253064e95843772 Mon Sep 17 00:00:00 2001
From: Richard Muzik <richard.muzik@turris.com>
Date: Mon, 16 Feb 2026 15:20:19 +0100
Subject: [PATCH] python-flask-httpauth: use correct wheel name

Fixes:

```
FileNotFoundError: [Errno 2] No such file or directory: '~/build/build_dir/target-arm_cortex-a9+vfpv3-d16_musl_eabi/pypi/Flask-HTTPAuth-4.8.0//openwrt-build/Flask_HTTPAuth-4.8.0-*.whl'
```
---
 lang/python/python-flask-httpauth/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lang/python/python-flask-httpauth/Makefile b/lang/python/python-flask-httpauth/Makefile
index 13fd50a50..aeec723ba 100644
--- a/lang/python/python-flask-httpauth/Makefile
+++ b/lang/python/python-flask-httpauth/Makefile
@@ -14,6 +14,8 @@ PKG_RELEASE:=1
 PYPI_NAME:=Flask-HTTPAuth
 PKG_HASH:=66568a05bc73942c65f1e2201ae746295816dc009edd84b482c44c758d75097a
 
+PYTHON3_PKG_WHEEL_NAME:=flask_httpauth
+
 PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec1@gmail.com>
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=LICENSE
-- 
2.53.0

